File: examples/LINUX/benchmarks/programs/fib.py
def fibonacci(n):
 a = 0
 b = 1
 nxt = 0
 for x in range(0, n):
     a = b
     b = nxt
     nxt = a + b
 return nxt

for i in range(100000):
 fibonacci(40)

print(fibonacci(40))

File: examples/LINUX/benchmarks/programs/fib.php
<?php

for($i = 0; $i < 100000; $i++)
  fibonacci(40);

echo fibonacci(40);

function fibonacci($a) {
  $b = 1;
  $c = 0;
  for($r = 0; $r < $a; $r++) {
    $a = $b;
    $b = $c;
    $c = $a + $b;
  }
  return $c;
}
